### PR TITLE
Add wandb.Audio support to Artifacts

### PR DESCRIPTION
**Pull request by davidwallacejackson**
Allows logging Audio objects in an Artifacts-friendly way and adds tests for same. Support for Audio in the Artifacts UI is up next.

### CODE REVIEW

File: standalone_tests/artifact_object_reference_test.py [hunk 1]
```diff
@@ -188,10 +188,22 @@ def _make_wandb_table():
 def _make_wandb_joinedtable():
     return wandb.JoinedTable(_make_wandb_table(), _make_wandb_table(), "id")
 
+def _make_wandb_audio():
```
Comment (tssweeney) [inline standalone_tests/artifact_object_reference_test.py:191]: Do you mind adding the Audio to the `_make_wandb_table` function (so basically add an entry to the `columns` array called "Audio", then add this `_make_wandb_audio()` entries to the data of `_make_wandb_table`?

**Review by tssweeney — CHANGES_REQUESTED** [review]
LGTM! Do you mind adding the Audio type to the table?

File: standalone_tests/artifact_object_reference_test.py (hunk 1 above)
Comment (davidwallacejackson) [reply standalone_tests/artifact_object_reference_test.py]: yep; done

**Review by tssweeney — APPROVED** [review]
:GTM - thanks for making those changes

**Pull request merged**